content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -69,11 +69,11 @@ Each `resources` array item is a separate config and contains a complete configu
 `resources.resources` field is an array of Kubernetes resource names (`resource` or `resource.group`
 that should be encrypted like Secrets, ConfigMaps, or other resources. 
 
-You can encrypt custom resources if you are running Kubernetes v1.26 or newer. 
-If custom resources are added to `EncryptionConfiguration` before v1.26, then after an upgrade
-to 1.26 and the CRD gets created, then it will be encrypted. If it already existed before that point, 
-a storage migration would be required to get all the CRs into an encrypted state just
-like any other built-in resource.
+If custom resources are added to `EncryptionConfiguration` and the cluster version is 1.26 or newer, 
+any newly created custom resources mentioned in the `EncryptionConfiguration` will be encrypted. 
```

Comment:
This, to me, seems repetitive because you're already mentioning that you're adding the custom resources to the `EncryptionConfiguration` once above. Is it intentional? The reason I ask is we're already clarifying below that any custom resources existing in etcd prior to that configuration will not be supported. We can make it concise for clarity & brevity by stating that, 

"Custom resources added to `EncryptionConfiguration` in clusters running Kubernetes versions 1.26 or newer will be encrypted by default."

WDYT? Does it take away from the meaning we intend to convey?

Reply:
Thanks for the review! Unfortunately shortening it does take away the intent that only newly created CRs will be encrypted.

Reply:
Okay, then we may leave it as-is.